src/main/resources/LanguageService.java

```diff
@@ -0,0 +1,26 @@
+package it.smartcommunitylab.aac.templates.service;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.Set;
+import java.util.TreeSet;
+
+import org.springframework.stereotype.Service;
+
+@Service
+public class LanguageService {
```

Comment:
You added this as a new file in resources. Please remove and edit the existing file in `src/main/java`